docs/SECURITY-OPERATIONS.md

```
@@ -0,0 +1,348 @@
+
+This is the answer to "Dependabot burst." See [scar in CLAUDE.md](#) — burst from `dependabot.yml` on aged repos.
+
+### 5.2 Enabling on a stale repo
+```yaml
+# In .github/dependabot.yml — set low limit initially
+open-pull-requests-limit: 1
+```
+Bump to 5+ once the initial burst is triaged.
+
+### 5.3 Security vs. version updates
+| Type | Trigger | Throttling |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)